autoblocks/_impl/prompts/manager.py

```diff
@@ -197,7 +198,7 @@ def exec(self) -> ContextManager[ExecutionContextType]:
         """
 
         @contextlib.contextmanager
-        def gen():
+        def gen():  # type: ignore
```

Comment:
the `exec` above is already incorrectly typed due to a pycharm bug, so just ignoring this inner context manager that it's wrapping